application/views/SurveysGroupsPermission/subviews/addUserGroupResult.php

```diff
@@ -1,13 +1,28 @@
 <div class="jumbotron">
     <h2 class="pagetitle h3"><?php eT('Add user group');?></h2>
     <?php if($result['warning']) : ?>
-        <p class="alert alert-warning"><?= $result['warning'] ?>
+    <?php
```

Comment:
Nice with widget instead of hard-coded HTML!

Reply:
Jaaaa!